examples/react-klevu-ui-v2/README.md

```diff
@@ -0,0 +1,27 @@
+# React + TypeScript + Vite
```

Comment:
I think this readme should contain information about this example. Reasonings why it exists and whats the point. I think we should underline it as a testing ground for UI react components.

Reply:
I added some information and some snippets for different examples. Let me know if it needs more work.